### PR TITLE
fetchOpenInterestHistory: adjust open interest structure

### PR DESCRIPTION
**Pull request by Dan-krm**
Adjusted the open interest structure across current exchanges that have the fetchOpenInterestHistory method.

Currently the structure is:
```
{
    symbol: 'BTC/USDT',
    baseVolume: 80872.801,
    quoteVolume: 3508262107.38,
    timestamp: 1649379000000,
    datetime: '2022-04-08T00:50:00.000Z',
    info: {
        symbol: 'BTCUSDT',
        sumOpenInterest: '80872.80100000',
        sumOpenInterestValue: '3508262107.38000000',
        timestamp: '1649379000000'
    }
}
```

This PR changes it to:
```
{
    symbol: 'BTC/USDT',
    openInterestAmount: 80872.801,
    openInterestValue: 3508262107.38,
    timestamp: 1649379000000,
    datetime: '2022-04-08T00:50:00.000Z',
    info: {
        symbol: 'BTCUSDT',
        sumOpenInterest: '80872.80100000',
        sumOpenInterestValue: '3508262107.38000000',
        timestamp: '1649379000000'
    }
}
```

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -4866,8 +4866,8 @@ module.exports = class bybit extends Exchange {
         const contractSize = this.safeString (market, 'contractSize');
         return {
             'symbol': this.safeSymbol (id),
-            'baseVolume': Precise.stringMul (numContracts, contractSize),
-            'quoteVolume': undefined,
+            'openInterestAmount': numContracts,
+            'openInterestValue': Precise.stringMul (numContracts, contractSize),
```
Comment (carlosmiei) [inline js/bybit.js:4870]: @Dan-krm we should be converting this to a number

File: js/binance.js [hunk 2]
```diff
@@ -6520,8 +6520,10 @@ module.exports = class binance extends Exchange {
         market = this.safeMarket (id, market);
         return {
             'symbol': this.safeSymbol (id),
-            'baseVolume': this.safeNumber (interest, 'sumOpenInterest'),
-            'quoteVolume': this.safeNumber (interest, 'sumOpenInterestValue'),
+            'baseVolume': this.safeNumber (interest, 'sumOpenInterest'),  // deprecated
```
Comment (carlosmiei) [inline js/binance.js:6523]: Can we avoid calling `safeNumber` twice with the same value?

File: js/bybit.js [hunk 3]
```diff
@@ -4866,8 +4866,10 @@ module.exports = class bybit extends Exchange {
         const contractSize = this.safeString (market, 'contractSize');
         return {
             'symbol': this.safeSymbol (id),
-            'baseVolume': Precise.stringMul (numContracts, contractSize),
-            'quoteVolume': undefined,
+            'baseVolume': Precise.stringMul (numContracts, contractSize),  // deprecated
```
Comment (carlosmiei) [inline js/bybit.js:4869]: We should be converting `baseVolume` to number here

File: js/huobi.js [hunk 4]
```diff
@@ -6686,8 +6686,10 @@ module.exports = class huobi extends Exchange {
         const timestamp = this.safeNumber (interest, 'ts');
         return {
             'symbol': this.safeString (market, 'symbol'),
-            'baseVolume': this.safeNumber (interest, 'volume'),
-            'quoteVolume': this.safeValue (interest, 'value'),
+            'baseVolume': this.safeNumber (interest, 'volume'),  // deprecated
```
Comment (carlosmiei) [inline js/huobi.js:6689]: same, avoid duplicated `safeNumber` calls pls

File: doc/manual.rst [hunk 5]
```diff
@@ -4490,8 +4490,10 @@ Open Interest Structure
 
    {
        symbol: 'BTC/USDT',
-       baseVolume: 80872.801,
-       quoteVolume: 3508262107.38,
+       baseVolume: 80872.801,  // deprecated
+       quoteVolume: 3508262107.38, // deprecated
+       openInterestAmount: 80872.801,
+       openInterestValue: 3508262107.38,
```
Comment (samgermain) [inline doc/manual.rst:4496]: These changes should be made to `wiki/manual.md`, not `doc/manual.rst`

**Review by samgermain — CHANGES_REQUESTED** [review]

**Comment by Dan-krm** [comment]
> **aax** and **ftx** should also be updated

**aax** and **ftx** only have fetchOpenInterest so there is nothing to update in this PR

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p bybit fetchOpenInterestHistory "BTC/USDT:USDT" 1h None 1 --sandbox
Python v3.10.6
CCXT v2.0.61
bybit.fetchOpenInterestHistory(BTC/USDT:USDT,1h,None,1)
[{'baseVolume': 86055.738,
  'datetime': '1970-01-20T06:56:49.200Z',
  'info': {'open_interest': '86055.738',
           'symbol': 'BTCUSDT',
           'timestamp': '1666609200'},
  'openInterestAmount': None,
  'openInterestValue': 86055.738,
  'quoteVolume': None,
  'symbol': 'BTC/USDT:USDT',
  'timestamp': 1666609200}]
```

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**